### DISCUSSION TITLE
Is it possible to drag a point across the graph?

### DISCUSSION THREAD

**Discussion by Fabiocss** [imaNNeo/fl_chart#1404] [Q&A]
Is it possible to drag a point of a line on the graph using the finger drag and drop gesture to the new position?

**Comment by freestyletime** [comment]
I have the same requirement but this sounds quite hard.

**Comment by imaNNeo** [comment]
It would be possible when we implement a way to map flutter positions to chart positions and vice versa
Follow #801
